File: Examples/Java/aspose-cloud-words-examples/src/main/java/com/aspose/words/cloud/convert/LoadWebDocumentExample.java
package com.aspose.words.cloud.convert;

import com.aspose.storage.api.StorageApi;
import com.aspose.words.api.WordsApi;
import com.aspose.words.cloud.config.Configuration;
import com.aspose.words.model.LoadWebDocumentData;
import com.aspose.words.model.SaveOptions;
import com.aspose.words.model.SaveResponse;

public class LoadWebDocumentExample {

    public static void main(String[] args) {
        // ExStart: 1
        try {
            // Instantiate Aspose Storage API SDK
            StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

            // Instantiate Aspose Words API SDK
            WordsApi wordsApi = new WordsApi(Configuration.apiKey, Configuration.appSID, true);


            SaveOptions saveOptionsData = new SaveOptions();
            saveOptionsData.setFileName("");
            saveOptionsData.setSaveFormat("doc");

            LoadWebDocumentData loadOtion = new LoadWebDocumentData();
            loadOtion.setLoadingDocumentUrl("http://www.aspose.com/corporate/default.aspx");
            loadOtion.setSaveOptions(saveOptionsData);

            // invoke Aspose.Words Cloud SDK API
            SaveResponse apiResponse = wordsApi.PostLoadWebDocument(loadOtion);

            if (apiResponse != null && apiResponse.getStatus().equals("OK")) {
                System.out.println("OK");
            }

        } catch (Exception e) {
            e.printStackTrace();
        }
        // ExEnd: 1

    }

}

File: Examples/PHP/src/Document/LoadWebDocumentExample.php
<?php
namespace Document;
require_once('../../vendor/autoload.php');

use Aspose\Storage\StorageApi;
use Aspose\Words\Configuration;
use Aspose\Words\WordsApi;
use Aspose\Words\Model\SaveOptionsData;
use Aspose\Words\Model\LoadWebDocumentData;
use Aspose\Words\Model\Requests\PostLoadWebDocumentRequest;
class LoadWebDocumentExample
{
    function main(){
        $config = new Configuration();
        $config->setAppKey('b125f13bf6b76ed81ee990142d841195')->setAppSid('78946fb4-3bd4-4d3e-b309-f9e2ff9ac6f9');
        
        $wordsAPI = new WordsApi(null, $config);
        $storageAPI = new StorageApi();
        $storageAPI->apiClient->apiKey ='b125f13bf6b76ed81ee990142d841195';
        $storageAPI->apiClient->appSid = '78946fb4-3bd4-4d3e-b309-f9e2ff9ac6f9';
        
        $saveOptions = new SaveOptionsData(array(
            "file_name" => "google.doc",
            "save_format" => "doc",
            "color_mode" => "1",
            "dml_effects_rendering_mode" => "1",
            "dml_rendering_mode" => "1",
            "update_sdt_content" => false,
            "zip_output" => false
        ));
        $body = new LoadWebDocumentData(array("loading_document_url" => "http://google.com", "save_options" => $saveOptions));
        $request = new PostLoadWebDocumentRequest($body);
        $result = $wordsAPI->postLoadWebDocument($request);
        echo $result;
    }
}

$document = new LoadWebDocumentExample();
$document->main();